app/assets/v2/js/base.js

```diff
@@ -230,18 +230,19 @@ const alertMessage = function(msg) {
   return html;
 };
 
-const _alert = function(msg, _class) {
+const _alert = function(msg, _class, remove_after_ms) {
```

Comment:
this is quite a nice little feature... removing the msg after some time... clears notifications so users dont have to!